combat-trainer.lic

```diff
@@ -1158,6 +1205,94 @@ class SpellProcess
   end
 end
 
+class PetProcess
+  include DRC
+
+  def initialize(settings)
+    echo('New Pet Process') if $debug_mode_ct
+
+    @zombie = settings.zombie
+    echo("  @zombie: #{@zombie}") if $debug_mode_ct
+    
+    @is_present = false
+    @current_stance = nil
+    @current_behavior = nil
+  end
+
+  def execute(game_state)
+    check_zombie
+    if game_state.dismiss_pet?
+      game_state.next_clean_up_step
+      dismiss_zombie
+      return true
+    end
+    false
+  end
+ 
+  def cfb_active?
+    DRSpells.active_spells.include?('Call from Beyond')
```

Comment:
This is also checked on line 425. Would it make sense to have a `GameState` helper for this check?